app/views/Mails/acceptrefuse/sendProposalApproved.scala.txt

```diff
@@ -53,13 +23,55 @@ About your speaker badge:
 =========================
 Your submission allows you to have a speaker badge which will give you access to the conference as a whole and to the speaker diner. No need to do anything from your side, your badge will be given to you right at the entrance at the beginning of day 1.
 
+Speaker's dinner:
+=========================
+All speakers will be invited to join the speaker's dinner that will take place the night before the conference on Monday 22th evening. More information will followed when we get closer to the event.
```

Comment:
"More information will follow as we get closer to the event"

Reply:
Done